DependencyInjection/SonataDashboardExtension.php

```diff
@@ -57,6 +58,9 @@ public function registerParameters(ContainerBuilder $container, array $config)
         $container->setParameter('sonata.dashboard.admin.block.entity', $config['class']['block']);
         $container->setParameter('sonata.dashboard.admin.dashboard.entity', $config['class']['dashboard']);
 
+        $container->setParameter('sonata.dashboard.admin.dashboard.templates.compose', $config['templates']['compose']);
+        $container->setParameter('sonata.dashboard.admin.dashboard.templates.compose_container_show', $config['templates']['compose_container_show']);
```

Comment:
There's no linebreak for these statements in the other SonataBundles (I try to be consistent with your usual styling)